data_management/models.py

```diff
@@ -667,16 +667,19 @@ class QualityControlled(BaseModel):
     ### Writable Fields:
     `object`: API URL of the associated `Object`
 
+    `field`: API URL of the `Object` representing the quality control document
+
     ### Read-only Fields:
     `url`: Reference to the instance of the `QualityControlled`, final integer is the `QualityControlled` id
 
     `last_updated`: Datetime that this record was last updated
 
     `updated_by`: Reference to the user that updated this record
     """
-    ADMIN_LIST_FIELDS = ('object',)
+    ADMIN_LIST_FIELDS = ('object', 'field')
 
     object = models.OneToOneField(Object, on_delete=models.PROTECT, related_name='quality_control')
+    field = models.OneToOneField(Object, on_delete=models.PROTECT, related_name='quality_control_document')
```

Comment:
Likewise, `document`?
```suggestion
    document = models.OneToOneField(Object, on_delete=models.PROTECT, related_name='quality_control_document')
```